etl/models/anti_recommendation_keys_by_key_tuple.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+
+from etl.models.types import RecordKey
+
+
+@dataclass(frozen=True)
+class AntiRecommendationKeysByKeyTuple:
```

Comment:
Discussed reworking this graph structure into (subject, object) tuples or (subject, objects) tuples.

Reply:
I have changed this into a tuple structure (`AntiRecommendationGraphTuple`).